### PR TITLE
Terms button covered by Scrolltotop button

### LINKED ISSUE #3928: ❗Important--Terms button not visible under mobile mode and also on some desktops.
### Description
Terms button is fully covered by go to top button. The user hence can't click on TERMS link.
Seen on mobile mode.

### Expected Behavior
The terms button must be visible.

### Screenshots
before

@Chadha93 I think the terms is a important thing for the user to know so should be solved quickly.
I have made some changes if you want I can do a PR or else if you have any suggestions.
It looks like this now.

after

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by parmishh**
**Description**

This PR fixes #3928 

**Notes for Reviewers**
fixes https://github.com/layer5io/layer5/issues/3928

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit bd7ad72522f2e4c0094590c4f8074a332fa0ac55 at: https://640a3a7960ff180088e65fa7--layer5.netlify.app

File: src/sections/General/Footer/footer.style.js [hunk 1]
```diff
@@ -232,6 +232,30 @@ const FooterWrapper = styled.section`
...
+               left: -28px;
+             }
+           }
+
+        @media screen and (max-width: 62rem) {
+           .status{
+              position: relative;
+              left: -28px;
+            }
+          }
```
Comment (Nikhil-Ladha) [inline src/sections/General/Footer/footer.style.js:257]: Could be aggregated under one media query.
Also, for media queries it is better to use `px` unit for resolutions over `rem`.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Thee issue is still not resolved. I could still see the `Terms` button being overlapped by the `Scroll-to-top` button for most of the screen resolutions after 1350px.

**Comment by parmishh** [comment]
@Nikhil-Ladha yes I got it working to fix for 1350px +. And also reducing code length. Will update it soon.

**Comment by parmishh** [comment]
@Nikhil-Ladha 
I have done some changes looking responsive on most devices mobile/desktop.
Have a look.

https://user-images.githubusercontent.com/91942072/224275073-d86e2072-7048-443b-9c8d-ead6317023c1.mp4

**Comment by parmishh** [comment]
@Nikhil-Ladha @Chadha93 please have a look.😊

**Comment by l5io** [comment]
🚀 Preview for commit aaf662149babb017520f779a1e24a1c0b966e182 at: https://640b08da0692467f9adc6378--layer5.netlify.app

File: src/sections/General/Footer/footer.style.js [hunk 2]
```diff
@@ -232,6 +232,33 @@ const FooterWrapper = styled.section`
...
+	     .privacy{
+               position: relative;
+               left: -28px;
+             }
+	     .status{
+              position: relative;
+              left: -28px;
+            }
+	     
+           }
```
Comment (Nikhil-Ladha) [inline src/sections/General/Footer/footer.style.js:253]: There is still an overlap in mobile view.
So, btter solution is reduce the width/height of the `scroll-to-top` button in mobile view and increase the bottom value such that the button lies in between the subscribe button the policies. And, increase the top margin a bit for the terms, status line.

**Comment by parmishh** [comment]
@Nikhil-Ladha thanks for suggestions I have changed the approach!
I have made the scrolltotop button smaller and given bottom so that it does not cover the terms button.
Have a look.
ll

**Comment by l5io** [comment]
🚀 Preview for commit fae6f564eb9b5dbce9a0effbb46dfc598990319d at: https://640b807341acc82523df9ef8--layer5.netlify.app

**Comment by Nikhil-Ladha** [comment]
The scroll-to-top button is now positioned incorrectly even for desktop view, along with that the size is reduced as well.
Those changes should be limited to mobile view only, probably after (leess than) a screen resolution of 600px or so. Please check that and update the code accordingly, and check for responsiveness before re-requesting for review and by that I mean do your due intelligence 🙏🏼

**Comment by Nikhil-Ladha** [comment]
And, also update the PR title to be something more meanigful and descriptive.

**Pull request closed without merging**